contracts/validatormanager/PoSValidatorManager.sol

```diff
@@ -73,12 +92,23 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         uint256 minimumStakeAmount,
         uint256 maximumStakeAmount,
         uint64 minimumStakeDuration,
+        uint256 minimumDelegationFee,
+        uint64 maximumStakeMultiplier,
         IRewardCalculator rewardCalculator
     ) internal onlyInitializing {
         PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+        require(minimumDelegationFee > 0, "PoSValidatorManager: zero delegation fee");
+        require(
+            minimumStakeAmount <= maximumStakeAmount,
+            "PoSValidatorManager: invalid stake amount range"
+        );
+        require(maximumStakeMultiplier > 0, "PoSValidatorManager: zero maximum stake multiplier");
```

Comment:
Should we support 0 as a valid value here? That would effectively disable any delegation, which could be a reasonable use case. 

I think we should enforce some maximum as well. If someone really wants to go above the maximum, it's easy to change, but otherwise could result in them unintentionally messing up the value used in the constructor. Just by gut feeling, I'd say a reasonable maximum `maximumStakeMultipier` to enforce would be 10.

Reply:
added maximum, since the multiplier is applied to inital stake amount the value of 1 already disables any further delegation, so 0 is not needed.